pkg/resources/fluentbit/config.go

```diff
@@ -42,6 +42,22 @@ var fluentBitConfigTemplate = `
     {{ $key }}  {{$value}}
     {{- end }}
     {{- end }}
+    {{- if .HealthCheck }}
+    {{- if .HealthCheck.Enabled }}
+    Health_Check On
+    {{- else }}
+    Health_Check Off
+    {{- end }}
```

Comment:
does it make sense to have a specific flag for this? wouldn't we want to enable healthcheck always if the healthcheck struct is defined?

Reply:
I just replicated the same config structure from fluent-bit but fair point. From a logging-operator user perspective this flag is not really needed.